core/views.py

```diff
@@ -727,6 +727,41 @@ def add_announcement(request):
             obj.body = form.cleaned_data['body']
             obj.save()
 
+            recievers = []
+            for user in User.objects.all():
+                recievers.append(user.email)
+
+            # send_mail(obj.title, obj.body, 'greekrhoverify@gmail.com', recievers) - can't use this function if we want to use bcc, but keep it in for now
+
+            truemessage = render_to_string('core/announcement_email.html', {
+                'user': request.user,
+                'body': form.cleaned_data['body'],
+            })
+
+
+
+            with get_connection(
+                host='smtp.gmail.com', 
+                port=587, 
+                username='announcementsgreekrho@gmail.com', 
+                password='greekrho1', 
+                use_tls=True
```

Comment:
These should be written in settings.py as environment variables to hide sensitive information.  Check out how I did the AWS variables at the bottom of the page.  The old gmail settings are actually in there still, if you want to just change those.  Then you can access the settings variables in this views file by using `from django.conf import settings` and `host = settings.HOST`, etc.